chart/datadog-operator/templates/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
           args:
             - --zap-level={{ .Values.logLevel }}
             - --zap-encoder=console
-            # - --zap-stacktrace-level=error // TODO: uncomment after releasing v0.3.0
+            # - --zap-stacktrace-level=panic // TODO: uncomment after releasing v0.3.0
```

Comment:
When will we remove the comment?

Reply:
When we release a new official Docker image (otherwise this parameter will break people using the chart with current official image)